src/main/resources/atlassian-plugin.xml

```diff
@@ -62,7 +62,12 @@
         <resource type="velocity" name="edit" location="/template/edit.vm"/>
         <resource type="velocity" name="view" location="/template/custom.vm"/>
     </customfield-type>
-
+    <customfield-type key="second-custom-field" name="Second custom field" i18n-name-key="second-custom-field.name"
+    class="com.atlassian.shele.shelePlugin.customfield.SecondCustomField">
+            <description key="second-custom-field.description"/>
+    <resource type="velocity" name="edit" location="/template/second.vm"/>
+        <resource type="velocity" name="view" location="/template/valid.vm"/>
```

Comment:
Names of your `.vm` are not describe their purpose. For excample you can create new folder for your custom field and rename files to `edit.vm` and `view.vm`